openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -13,6 +13,32 @@
 _logger = logging.getLogger(__name__)
 
 
+def login_or_registration_required_at_checkout(cr):
+    """ website_sale_require_login merged into website_sale. Check if the
+    website_sale_require_login module was installed in v15 to set the
+    website.account_on_checkout field as mandatory so that the functionality
+    remains the same, login/registration required for checkout. """
```

Comment:
```suggestion
    """ The website_sale_require_login module is merged into website_sale. Check if the
    it was installed in v15 to set the website.account_on_checkout field as mandatory so that
    the functionality remains the same, login/registration required for checkout. """
```